### ISSUE TITLE
MAML Forward Function

### ISSUE THREAD

**Issue by reasonW** [Vision-CAIR/cs326-few-shot-classification#2]
line 34 in maml.py:
`    def __call__(self, x: Tensor) -> Tensor:
`
Should this function be changed to
    `def __call__(self, x: Tensor, params: Tensor = None) -> Tensor:`

**Comment by reem-codes** [comment]
I've changed it like this too and it's easier than something like `model.target_model(...)`

**Comment by universome** [comment]
Yes, you can either change it like that or you can use `model.target_model(...)` as suggested by Reem
